bouncer.go

```diff
@@ -444,3 +468,51 @@ func crowdsecQuery(bouncer *Bouncer, stringURL string, isPost bool) ([]byte, err
 	}
 	return body, nil
 }
+
+func appsecQuery(bouncer *Bouncer, ip string, httpReq *http.Request) error {
+	routeURL := url.URL{
+		Scheme: bouncer.appsecScheme,
+		Host:   bouncer.appsecHost,
+		Path:   "/",
+	}
+	var req *http.Request
+	if httpReq.Body != nil && httpReq.ContentLength > 0 {
+		bodyBytes, err := io.ReadAll(httpReq.Body)
+		if err != nil {
+			return fmt.Errorf("appsecQuery:GetBody %w", err)
+		}
+		httpReq.Body = io.NopCloser(bytes.NewBuffer(bodyBytes))
```

Comment:
We might need to be carefull here, because if the body is super large, could it crash the plugin or use a lot of memory ?

Reply:
Some take in "Modsecurity" is to be able to only process the first XX bytes from the body if it is very large. For instance, in front of video/drive website protected

Reply:
https://github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin/issues/125